docs/Coding-Guidelines/HybridCRT.md

```diff
@@ -125,22 +123,58 @@ repository root imports
 all projects in the repository via this statement:
 
 ```xml
-  <!-- Build with the hybrid CRT (Universal CRT + Static VS CRT (for what little the Universal CRT doesn't cover) -->
+  <!-- Build with the Hybrid CRT (Universal CRT + Static VS CRT (for what little the Universal CRT doesn't cover) -->
   <Import Project="$(MSBuildThisFileDirectory)HybridCRT.props" />
 ```
 
-This applies to all projects in the repository (product, test, ...).
+## Hybrid CRT for End Developers
 
 ### Hybrid CRT C/C++ Apps
 
-For their own CRT usage, most C/C++ app developers will continue to link either statically, 
-with no redistribution requirements, or dynamically, requiring either a Microsoft.VCLibs 
-dependency (for packaged apps) or a VCRedist dependency (for unpackaged apps). 
-For unpackaged self-contained C/C++ Windows App SDK apps, some developers may also find 
+For their own CRT usage, C/C++ Windows App SDK app developers may continue to link statically,
+with no redistribution requirements, or dynamically, requiring either a VCLibs*
+dependency (for packaged apps) or a vc_redist.*.exe dependency (for unpackaged apps).
+
+For unpackaged self-contained C/C++ Windows App SDK apps, developers may find
 Hybrid CRT linkage useful for enabling xcopy deployment. For these apps, the Directory.Build.*
 files described above can be included in the project. For apps that also include C++/CX code,
 additional considerations are described below.
 
+### Hybrid CRT C/C++ Libs
+
+Existing libraries targeting Universal Windows (UWP) may be consumed by a Windows App SDK app,
+provided:
+
+* The app references [VCRT Forwarders](https://github.com/microsoft/vcrt-forwarders),
+to map App CRT linkage to Desktop CRT (vcruntime140_app.dll -> vcruntime140.dll ...), or
+* The library project includes this property, to force Desktop CRT linkage (vcruntime140.dll ...):
+
+  ```xml
```

Comment:
NIT: Unindent to column 1

looks like simple editing error

Reply:
indented intentionally to align with parent bullet